kong/pdk/request.lua

```diff
@@ -568,10 +569,7 @@ local function new(self)
       error("header name must be a string", 2)
     end
 
-    local header_value = _REQUEST.get_headers()[name]
-    if type(header_value) == "table" then
-      return header_value[1]
-    end
+    local header_value = var["http_" .. lower(name:gsub("-", "_"))]
```

Comment:
When I view the lua-nginx-module's code, I found ngx-lua has done the str_lower job, I think we can remove the call of lua `lower`.

You can do a simple test for it.

Reply:
You're right.